<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KonnectionMonitorClass</class>
 <widget class="QMainWindow" name="KonnectionMonitorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KonnectionMonitor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="refreshButton">
      <property name="toolTip">
       <string>Refresh</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../res/konnectionmonitor.qrc">
        <normaloff>:/icon/refresh24.png</normaloff>:/icon/refresh24.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="autoRefreshCB">
      <property name="text">
       <string>every</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QSpinBox" name="autoRefreshSB">
      <property name="correctionMode">
       <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
      </property>
      <property name="suffix">
       <string> msecs</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>60000</number>
      </property>
      <property name="singleStep">
       <number>500</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="4">
     <widget class="QLineEdit" name="filterLE">
      <property name="toolTip">
       <string>filter connections</string>
      </property>
      <property name="placeholderText">
       <string>Regexp filter..</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="rootModeButton">
      <property name="toolTip">
       <string>Root mode</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../res/konnectionmonitor.qrc">
        <normaloff>:/icon/lock24.png</normaloff>
        <disabledoff>:/icon/unlock24.png</disabledoff>:/icon/lock24.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QPushButton" name="aboutButton">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="icon">
       <iconset resource="../res/konnectionmonitor.qrc">
        <normaloff>:/icon/info24.png</normaloff>:/icon/info24.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="7">
     <widget class="QTableView" name="connectionTableView">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>702</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../res/konnectionmonitor.qrc"/>
 </resources>
 <connections/>
</ui>
